int32[6] motor_speed